src/components/icons/IconInProgress.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @flow
+ * @file Icon
+ * @author Box
+ */
+
+import React from 'react';
+import IconCross from './IconCross';
+import LoadingIndicator from '../LoadingIndicator';
+import type { IconType } from '../../flowTypes';
+
+const IconInProgress = ({ color = '#222', className = '', width = 14, height = 14 }: IconType) =>
```

Comment:
Move this file to the location where its being used, icons directory is for SVGs only. Will be removing it once we can reference box-react-ui. It should also ideally be next to the file that has its CSS class.